chainer/functions/math/clip.py

```diff
@@ -10,8 +10,11 @@ class Clip(function_node.FunctionNode):
     """Clips (limits) elements of input variable."""
 
     def __init__(self, x_min, x_max):
+        if (x_min is None) and (x_max is None):
+            raise ValueError('must set either max or min')
+
         # x_min must be less than x_max.
-        if x_min >= x_max:
+        if (x_min is not None) and (x_max is not None) and (x_min >= x_max):
```

Comment:
I think you can skip the parenthesis here too. But maybe this is more readable. What do you think?

Reply:
I think `None` checks should be separate `if` statement for readability.
Is that OK in Chainer Python style?

Reply:
That's fine too.